### ISSUE TITLE
Missing '' around output dir in HTML reports

### ISSUE THREAD

**Issue by maasha** [maasha/BioDSL#17]
Possible also in any verbose output:

```
run(progress: true, verbose: false, output_dir: Jakub_Result-2015-03-24, report: p6.html)
```

should be

```
run(progress: true, verbose: false, output_dir: 'Jakub_Result-2015-03-24', report: p6.html))
```

**Comment by maasha** [comment]
and '' around report as well.

**Comment by maasha** [comment]
fixed: e0743ccd93cfa1435ca5d172779b3552af882e1a
